docs/source/legacy/datasets.rst

```diff
@@ -1,69 +0,0 @@
-torchtext.legacy.datasets
-====================
-
-.. currentmodule:: torchtext.legacy.datasets
```

Comment:
Is the documentation for older datasets completely gone? (unless `docs/source/examples.rst` covers this?) Is that intentional?

Reply:
There won't be any datasets in the legacy folder. The new datasets are not in experimental folder and the old datasets are in the original place. So I delete the legacy folder.